validation/src/shared_table/mod.rs

```diff
@@ -284,6 +284,7 @@ impl<Fetch: Future> ParachainWork<Fetch> {
 				&*api,
 				id,
 				collation,
+				None,
```

Comment:
@rphmeier Could you please advise how it would be better to pass value here?

Reply:
it could be a parameter of whatever's instantiating `ParachainWork`.